usaspending_api/download/lookups.py

```diff
@@ -65,6 +70,7 @@
         "assistance_data": "transaction__assistance_data",
         "filter_function": universal_transaction_matview_filter,
         "is_for_idv": False,
+        "annotations_function": universal_transaction_matview_annotations,
```

Comment:
What if someone (possibly me) needs to add a second annotation for a different field of the same object?

Reply:
Do you mean having one for assistance_data and one for contract_data, or having the function return two annotations?

Reply:
Nevermind, I failed to realize that these functions themselves distinguish between fields.